src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -52,6 +52,10 @@
     <Compile Include="$(CommonPath)\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
+    <ProjectReference Include="..\..\System.Net.Http\pkg\System.Net.Http.pkgproj">
```

Comment:
You cannot reference a pkgproj from a source project.  What is it you are trying to do here, you don't need a new reference since (hopefully) you aren't adding any API.